client/src/main/scala/geotrellis/admin/client/routes/GeotrellisAdminViewer.scala

```diff
@@ -39,7 +40,7 @@ object GeotrellisAdminViewer {
             "Metadata JSON"
           ),
           (if (state.showJSON) {
-            <.code("This is code")
+            props().displayM.rawMetadata.render(j => <.pre(j))
```

Comment:
I feel like this is probably a fine way to handle it until more functionality is required